src/Sylius/Bundle/CoreBundle/Doctrine/ORM/OrderRepository.php

```diff
@@ -39,6 +39,18 @@ public function __construct(EntityManager $entityManager, ClassMetadata $class)
         $this->associationHydrator = new AssociationHydrator($entityManager, $class);
     }
 
+    public function getOrderById(string $id): ?OrderInterface
```

Comment:
as we are allowing for the null result we should adjust the naming. What is more, it could go to Order namespace, is suppose 
```suggestion
    public function findOrderById(string $id): ?OrderInterface
```

Reply:
I've renamed it.